packages/server/src/modules/mail/templates/en-US/invite-user.ejs

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+  <body>
```

Comment:
Yes, it was like that all along for all emails, starting from the project's creation. The email clients seem to silently parse the incorrect markup and fix it. It's not the only thing that an email markup misses, though:

https://www.htmlemailcheck.com/
![image](https://github.com/ilmercatinodellibro/ilmercatinodellibro/assets/6266078/a3abaedc-71eb-40d5-a23f-286f9ef28981)

But, on the other hand, Mailtrap says the markup is fine:
![image](https://github.com/ilmercatinodellibro/ilmercatinodellibro/assets/6266078/2ba9758b-abf4-40f7-90d0-f9547e14706f)
and the markup seems to be getting filled with the missing stuff:
![image](https://github.com/ilmercatinodellibro/ilmercatinodellibro/assets/6266078/95594485-eba0-4805-b436-49c803935af3)
it's either Mailtrap's email-receiving logic auto-fixes the stuff, or the emailer library is filling in the gaps automatically. I'd say it's the emailer library doing it but through the template compilation step. In either case, we should write a correct markup to be on the safe side.

We should also probably look into these:
https://nodemailer.com/message/addresses/ - use an address object to also pass in the user's first name and last name along with the email, also see below image regarding spam analysis
https://nodemailer.com/message/list-headers/ - not a concern now as we only do transactional emails, but it would be good to check for mailing lists in case we will have them in the future

![image](https://github.com/ilmercatinodellibro/ilmercatinodellibro/assets/6266078/01833a91-d726-4323-ba48-ee5d1f0f5720)
`TO_NO_BRKTS_HTML_ONLY` here is there for two reasons:
- no brackets: `to` is like `user@example.com` instead of `Test User <user@example.com>` or even `user@example.com <user@example.com>`. We can easily solve this as mentioned in the links above.
- HTML only: no plain text fallback. We can use https://github.com/andris9/nodemailer-html-to-text to easily address this.

Reply:
I see! Nice and detailed explanation, thanks!

First thing: at this point I say we address this into a dedicated PR+its task, because it seems like this fix here can be a little bit bigger and we should not take care of them at once together.

1. Interesting that probably the library automatically fixes some of the things it sends.
2. `no brackets: to is like user@example.com instead of Test User <user@example.com> or even user@example.com <user@example.com>. We can easily solve this as mentioned in the links above.` I agree about this one.
3. `HTML only: no plain text fallback. We can use https://github.com/andris9/nodemailer-html-to-text to easily address this` -> that library seems a little bit outdated. Are we sure it is safe to use it?

So if you agree, we can open another task related to this one a place there these things

Reply:
> that library seems a little bit outdated. Are we sure it is safe to use it?

Not sure, but I found it through Nodemailer's documentation. I will check if it's usable, whether there are better alternatives, and whether it's simple/worth implementing it ourselves.